Add module to extract feed to external OBS system.

lostcarpark opened the pull request:
This change adds a module to extract a JSON file for each room that can be imported to an external OBS system, to allow program schedule to be displayed on projectors and on virtual attendee feeds between sessions.

bcholmes commented:
Looks good.

bcholmes commented on webpages/module/obs/feeds.php:
```diff
@@ -0,0 +1,116 @@
+require_once __DIR__ . '/../../error_functions.php';
+require_once __DIR__ . '/../../render_functions.php';
+require_once __DIR__ . '/../../StaffCommonCode.php';
+require_once 'obs_functions.php';
```
No __DIR__ ?

lostcarpark replied:
You're right. I didn't have __DIR__ because it's in the same directory. I should have used __DIR__, just without the .. parent directory references.

bcholmes replied:
Were you planning on changing that, or just leave it as is?

lostcarpark replied:
Yes, I'm planning to change this, and a couple of other bits in line with the module changes.

bcholmes replied:
👍

lostcarpark commented:
Updated to inherit ModuleBase class.

Moved ObsFeedModule into Obs Namespace, and into obs directory.

Corrected missing __DIR__ in feeds.php.

This is another one that I would like to add into a generalised cron process for modules, but not sure when I will have that ready.
